api/src/main/java/bio/terra/tanagra/service/search/AttributeVariable.java

```diff
@@ -0,0 +1,25 @@
+package bio.terra.tanagra.service.search;
+
+import com.google.auto.value.AutoValue;
+
+/**
+ * An {@link Attribute} for a bound {@link Variable}.
+ *
+ * <p>e.g. "x.attribute" in "SELECT x.attribute FROM entity as x".
+ */
+@AutoValue
+public abstract class AttributeVariable {
```

Comment:
Is "variable" used here as a Tanagra term, or a SQL term? I'd thought this maps more closely to something like a "select expression", though perhaps more narrowly constrained than that to support attributes.

Reply:
How about something like AttributeReference or ReferencedAttribute?